components/log-viewer-webui/client/src/ui/MainLayout.tsx

```diff
@@ -31,7 +35,7 @@ const MainLayout = () => {
     const [collapsed, setCollapsed] = useState(false);
 
     return (
-        <Layout className={"main-layout"}>
+        <Layout className={styles["main-layout"]}>
```

Comment:
Let's use camelCasing for class names in CSS modules, following the recommendation at https://github.com/css-modules/css-modules/blob/master/docs/naming.md

Then we can use the dot-notation here

Reply:
Let me quickly check the linter on my end.

Reply:
> https://github.com/mrmckeb/typescript-plugin-css-modules

this plugin is only a Language Service Plugin, which means it only have effects in IDEs (than `tsc -b`).

---

How about this:
1. We always name the classes in camelCase in CSS modules.
2. If we want to use dot-notation access, we shall disable `noPropertyAccessFromIndexSignature` in `tsconfig.app.json`.
3. Alternatively, we can update `eslint.config.mjs` to add below block:
```js
{
     files: [
         "**/*.ts",
         "**/*.tsx",
     ],
     rules: {
         "dot-notation": "off",
         "@typescript-eslint/dot-notation": "error",
     },
 },

```
I will also be adding the above to `eslint-config-yscope` so the config lib users do not have to config those explicitly.

Let me know what you think.